lib/index.js

```diff
@@ -345,19 +345,19 @@ setup_brush = function(graph){
           .attr("transform", function(d, i) { return "translate(0,"+ i * 20 +")"; });
 
       legend.append("rect")
-          .attr("x", width + 20)
```

Comment:
In the old version, the pca plot and the legend are in the same svg. "width" is the width for pca plot. The legend is located at the position (width+20).